pkg/deploy/strategy/recreate/recreate.go

```diff
@@ -106,13 +106,11 @@ func (s *RecreateDeploymentStrategy) Deploy(deployment *kapi.ReplicationControll
 
 	// Execute any post-hook. Errors are logged and ignored.
 	if params != nil && params.Post != nil {
-		// TODO: handle this in defaulting/conversion/validation?
-		if params.Post.FailurePolicy == deployapi.LifecycleHookFailurePolicyAbort {
-			params.Post.FailurePolicy = deployapi.LifecycleHookFailurePolicyIgnore
-		}
-		err := s.hookExecutor.Execute(params.Post, deployment, s.retryPeriod)
+		err := s.hookExecutor.Execute(params.Post, deployment, "posthook")
 		if err != nil {
-			glog.Infof("Post hook failed: %s", err)
+			glog.Errorf("Post hook failed: %s", err)
```

Comment:
This isn't infrastructure code, it's a binary run in a docker container that the user should be able to view the output of... my understanding is util.HandleError is for us to inject error handling behavior for infra components. There is no time we'd ever inject different behavior here.